File: functions/update_finantial_cal.py
#samantha naranjo, finantial cal update
print("Welcome to my finantial calculator, Please put in your information")

def info(cost, income, type):
    percentage= cost/income *100
    print(f"your {type} is ${cost:.2f} rent which is,{percentage}%, of your income")

def user(type):
    quesition = "what is your monthly " + type + "?\n"
    return float(input(quesition))
income = user("income")
rent = user("rent")
utilities= user("utilities")
groceries= user("groceries")
transportation= user("transportation")
spending= rent+utilities+groceries+transportation
savings = income*0.1

info(rent, income, "rent")
info(utilities, income, "utilites")
info(groceries, income, "groceries")
info(transportation, income, "transportation")
info(rent, income, "rent")
info(savings, income, "savings")
info(spending,income, "spending")

File: functions/update_finantial_cal.c
//samantha naranjo, update fintantial cal
#include <stdio.h>

float get_user_input(char *prompt) {
    float value;
    printf("%s", prompt);
    scanf("%f", &value);
    return value;
}
void print_result(char *item_name, float amount, float income) {
    float percentage = (amount / income) * 100;
    printf("Your %s are $%.2f which is %.2f%% of your income.\n", item_name, amount, percentage);
}

int main() {
    float income, rent, utilities, groceries, transport, savings, spendings;

    income = get_user_input("What is your income?\n");
    rent = get_user_input("How much is your rent?\n");
    utilities = get_user_input("How much are your utilities?\n");
    groceries = get_user_input("How much are your groceries?\n");
    transport = get_user_input("How much is your transportation?\n");

    savings = income * 0.1;
    spendings = income - savings - rent - utilities - groceries - transport;

    printf("Welcome to my calculator that will help you manage your monthly finances!\n");

    print_result("rent", rent, income);
    print_result("utilities", utilities, income);
    print_result("groceries", groceries, income);
    print_result("transportation", transport, income);
    print_result("savings", savings, income);
    print_result("spendings", spendings, income);

    return 0;
}

